src/app/api/v1/user/email/route.ts

```diff
@@ -0,0 +1,137 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getToken } from "next-auth/jwt";
+import { NextRequest, NextResponse } from "next/server";
+import AppConstants from "../../../../../appConstants";
```

Comment:
semi-related but definitely non-blocking: I wonder if we could use something like [path mapping](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) (or https://www.typescriptlang.org/tsconfig/#paths or https://medium.com/slackernoon/use-typescript-aliases-to-clean-up-your-import-statements-7210b7ec2af1) to shorten these imports and remove the deeply nested "../../" paths.

Reply:
@pdehaan that looks really cool - we should have a separate PR to introduce it and convert everything over though. I'd like input from front-end folks like @flozia and @Vinnl on whether they think it's a good thing for us to use. I'm not really familiar with it but sounds promising, thanks!

Reply:
I would like to do something about our import statements for sure, it's kind of a papercut when refactoring.

Reply:
> as more of our code is TypeScript, editors can more easily suggest the right auto-imports, as well as update them when you move files around.

This auto-magic™ is really helpful. I don't stress about paths bc of them.

Reply:
I agree with @Vinnl's points above. I think the issue will resolve itself after our Nextjs migration, and overtime as we become more disciplined with our types.